apps/pdl-forvalter/src/main/java/no/nav/pdl/forvalter/consumer/GeografiskeKodeverkConsumer.java

```diff
@@ -76,7 +76,7 @@ public String getTilfeldigLand() {
     public String getPoststedNavn(String postnummer) {
         return tokenExchange
                 .exchange(properties)
-                .flatMapMany(token -> new GeografiskeKodeverkCommand(webClient, POSTNUMMER_URL, postnummer, token.getTokenValue()).call())
+                .flatMapMany(token -> new GeografiskeKodeverkCommand(webClient, POSTNUMMER_URL, "postnummer=" + postnummer, token.getTokenValue()).call())
```

Comment:
Betyr dette at postnavnOppslag heller ikke virker ? 😟

Reply:
Ja, så at den kun ble brukt av KontaktInfo for dødsbo mapper. Så jeg bestilte en kontakinfo i Dolly med spesifikk postnummer, men fikk ikke det jeg ba om